fala/apps/adviser/utils.py

```diff
@@ -0,0 +1,46 @@
+# utils.py
+
+CATEGORY_TRANSLATIONS = {
```

Comment:
This CATEGORY_TRANSLATIONS can be moved out into it owns python file. 

Let's make a new directory called constants under apps, so `fala/apps/constants`

Here we'll make a python file call `categories.py`

Reply:
Don't forget to place an `__init__.py` into the directory. It helps with imports with Python.